lib/bolt/cli.rb

```diff
@@ -390,6 +390,14 @@ def handle_parser_errors
     def execute(options)
       if options[:mode] == 'plan' || options[:mode] == 'task'
         begin
+          if Gem.win_platform?
```

Comment:
I also tried this with `bolt command` by moving this to the beginning of the method.

resulted in speed up as well

Connection times in seconds
```
Command
----
8.9 - 8.9 - 9.0

to

3.9 - 4.4 - 3.8
```